extensions/yaml_support/labgraph_monitor/labgraph_monitor.py

```diff
@@ -0,0 +1,29 @@
+import labgraph as lg
+from .generate_lg_monitor.generate_lg_monitor import generate_graph_topology, set_graph_topology
+from .server.lg_monitor_server import run_topology
+
+class LabgraphMonitor:
```

Comment:
LGTM, Ig we could update the docs to use this interface instead.

Reply:
Sounds good!